background/test/app.test.js

```diff
@@ -10,32 +10,45 @@ const app = `https://testservice-dot-${projectId}.${regionId}.r.appspot.com`;
 const {expect} = require('chai');
 const {v4: uuidv4} = require('uuid');
 
-//in kokoro, need to set project id, auth, and region ID
-//use gcloud and firebase cli
-//how to get cleanup to wait
 describe('behavior of cloud function', function() {
   this.timeout(240000);
   const uniqueID = uuidv4().split('-')[0];
   before(() => {
     cp.execSync(`npm install`, {cwd: path.join(__dirname, '../', 'function')});
-    cp.execSync(
-      `gcloud functions deploy translate-${uniqueID} --allow-unauthenticated --set-env-vars=unique_id=${uniqueID} --runtime nodejs8 --trigger-topic translate`,
-      {cwd: path.join(__dirname, '../', 'function')}
-    );
-    cp.execSync(`gcloud app deploy`, {
-      cwd: path.join(__dirname, '../', 'server'),
-    });
+    try {
+      cp.execSync(
+        `gcloud functions deploy translate-${uniqueID} --allow-unauthenticated --set-env-vars=unique_id=${uniqueID} --runtime nodejs8 --trigger-topic translate`,
+        {cwd: path.join(__dirname, '../', 'function')}
+      );
+    } catch (err) {
+      console.log("wasn't able to deploy google cloud function");
```

Comment:
Maybe make the case match on this log and the one below? Sorry, verrry nitpick :)

Reply:
No problem! Thanks for being so detail-oriented, sorry for it being sloppy :)